Objects/typeobject.c

```diff
@@ -11285,6 +11294,83 @@ update_all_slots(PyTypeObject* type)
     }
 }
 
+int
+_PyType_InitSlotDefsNameCounts(PyInterpreterState *interp)
```

Comment:
I suggest a shorter function name:

```suggestion
_PyType_InitSlotDefs(PyInterpreterState *interp)
```

You may add a comment explaining the purpose of this function.

Reply:
Thanks! I agree that original name too long and not very clear.